crates/bevy_app/src/app_builder.rs

```diff
@@ -368,10 +410,9 @@ impl AppBuilder {
         self
     }
 
-    /// Initialize a resource in the current [App], if it does not exist yet
+    /// Initialize a resource in the current [`App`], if it does not exist yet
     ///
-    /// Adds a resource that implements `Default` or [`FromResources`] trait.
-    /// If the resource already exists, `init_resource` does nothing.
```

Comment:
I think this behavior is important to call out, as insert_resource does not behave this way.

Reply:
You mean the trait implementation or the do-nothing clause? Because in the PR diff the same line of the first one (the trait impl) is added afterwards.

I actually discussed with @MinerSebas about removing the do-nothing clause in this kind of situations since it doesn't actually add new information compared to the first line.

Reply:
Do nothing clause. I think the contrasting behavior with insert_resource is surprising and important.

Reply:
Ok, fixed 👍🏻